package.json

```diff
@@ -42,11 +42,13 @@
     }
   },
   "devDependencies": {
+    "@babel/core": "~7.2.2",
     "@magento/directive-parser": "~0.1.1",
-    "@magento/eslint-config": "~1.2.3",
+    "@magento/eslint-config": "~1.3.0",
     "@storybook/addon-actions": "~3.4.2",
     "@storybook/addons": "~3.4.6",
     "@storybook/react": "~4.0.11",
+    "acorn": "~6.0.5",
```

Comment:
What is using acorn? I couldn't find any existing code or changes in this PR that indicate its use.

Reply:
It's a peer dependency of a couple of eslint plugins. NPM was complaining.